### DISCUSSION TITLE
extended ascii / UTF8 in ctrlr info/alertWindows?

### DISCUSSION THREAD

**Discussion by fbit-itc** [RomanKubiak/ctrlr#579] [Q&A]
Hello everybody
has anyone a solution how to use non-standard ascii chars (like french accents) in the ctrlr (juce)  info/alertWindows?
Thx
Frank

**Comment by dobo365** [comment]
Interesting question so I took some time to find it out ;-)
Ctrlr 5.3.201 is using Lua 5.1
You can use string.char(code) where code is any ASCII code. 
You will find éèçà and co around 230.

**Reply by fbit-itc** [reply]
good idea, thanks, seems, gsub-ing by string.char() is the recommended workaround for lua - not the most elegant solution tough (as many things in lua ; ) 

Looked at it closer: ctrlr obviously uses utf-8.
using non-standards in modulator strings (name, text)  is correctly displayed 
using non-standards in utils.XXXWindow/AlertWindow class or console output displays in iso (= utf-iso mixed pickles) 

Well, highly appreciated standard issue in IT .... programmers that code on Win PC for later deployment into Unix environments ;)
